docs/configuration/tls-ingress.md

```diff
@@ -0,0 +1,60 @@
+# HTTPS using LetsEncrypt
```

Comment:
This is just a cleanup of `docs/configuration/tls-ingress-aws.md`, but github can't pick up on that. I moved the EKS specific settings to the EKS quickstart and cleaned up a bit to reflect the new configuration.